src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -756,9 +796,11 @@ def test_tenant_level_deployment(self):
         curr_dir = os.path.dirname(os.path.realpath(__file__))
         self.kwargs.update({
             'tf': os.path.join(curr_dir, 'tenant_level_template.json').replace('\\', '\\\\'),
+            'ts': '/subscriptions/a1bfa635-f2bf-42f1-86b5-848c674fc321/resourceGroups/TemplateSpecsCLI/providers/Microsoft.Resources/TemplateSpecs/BasicTemplateSpecs/versions/1.1.0',
```

Comment:
Same as above

Reply:
The resource cannot yet be dynamically created. We need to add the template-spec command module (CRUD operations) before the test cases can be updated.